src/pages/options/Content/OptionsCard/OptionsList/OptionItem/OptionWidgets/CustomCommands/index.js

```diff
@@ -0,0 +1,45 @@
+import { Component, h } from 'preact'
+import TextArea from '../TextArea'
+
+const DEFAULT = {
+  source: 'document.body.style.backgroundColor = "blue"'
+}
+
+class CustomCommand extends Component {
+  render ({ command, onChange }) {
+    return (
+      <li>
+        <p>This is a command</p>
+        <TextArea
```

Comment:
So, Widgets are meant to be standalone containers representing a single option. Although code reuse is nice, I think it's important to be able to change the TextArea Widget independently of the CustomCommand Widget.